pages/common/moro.md

```diff
@@ -1,23 +1,31 @@
-# Moro
+# moro
 
 > Track work time.
 
-- Run moro once right after starting the working day, and once more before leaving work:
+- Invoke `moro` without parameters, to set the current time as the start of the working day:
```

Comment:
Don't think that comma is needed.

Reply:
It isn't needed in the sense that the sentence would be incorrect without it, but I think it eases the reading flow, by marking semantic pauses in long sentences like this one.

Otherwise the reader has to parse the entire sentence to understand its structure and break it down into its major components (clauses).

Reply:
True, let's leave it in then.